src/components/App.tsx

```diff
@@ -31,38 +45,63 @@ class App extends React.Component<RouteProps, State> {
     return (
       <div className="App">
         <header className="App-header">
-          <h1 className="App-title">{this.state.headerText}</h1>
+          <h1 className="App-title">
+            {this.getHeaderText(this.state, this.logName)}
+          </h1>
         </header>
-        <LiftLogContainer
-          entries={this.state.logEntries}
-          onAddEntry={(entry: LiftLogEntry) => this.handleAddEntry(entry)}
-        />
+        {this.state.isLoading || this.state.networkErrorOccured ? null : (
```

Comment:
In order to not repeat `this.state...` in many places in the component's `render` you can destructure `state` before `return`:
```
const { isLoading, networkErrorOccurred } = this.state;
```

Reply:
Oops, I somehow missed this comment - will include in the next PR.